CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ Changelog
 
 # 2.x release
 
+## v2.6.2
+
+- Fix: used full filename for main in package.json
+- Other: pinned all dev dependencies (codecov -> teeny-request had on breaking change)
```

Comment:
instead of pinning all dev dependencies, maybe we can just remove codecov from `2.x`?

Reply:
As long as codecov works then we could still use it? it's going to stop working Feb 1 2022... maybe remove it then?

I did a smaller pining earlier where i only pinned codecov (+ that i added & pinned teeny-request into devDep also) it seemed to work fine. Should we do this instead?

Reply:
I removed codecov a long time ago because it produced to many random failed builds due to 5xx errors. 

We can leave it in the `main` branch, I don't see much value for it in `2.x`. 

My recommendation would be to get to 100% coverage and then fail the tests if it drops below, no need for an external service. That's what I do with my projects anyway, no more random CI errors when the coverage service is down

> I did a smaller pining earlier where i only pinned codecov (+ that i added & pinned teeny-request into devDep also) it seemed to work fine. that's when i got past the error and starting getting errors with rollup instead
> Should we do this instead?

I would prefer that

Reply:
Right now in v3 we use c8 instead of codecov that uses nodes internal code coverage instead of wrapping all code to check if some stuff are executed + c8 runs locally with the CI workflow
So codecov is removed in v3.

> I would prefer that

Then I will do that.

Reply:
fixed